From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Mark D Horn <mark.d.horn@intel.com>
Date: Tue, 23 Feb 2021 14:17:39 -0800
Subject: [PATCH] Fix tools scripts permissions

Clear Linux OS does not leave group write permissions enabled
unless explicitly needed for operational reasons.

Signed-off-by: Mark D Horn <mark.d.horn@intel.com>
---
 tools/meson.build | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/tools/meson.build b/tools/meson.build
index ac714e6..ed8cfbd 100644
--- a/tools/meson.build
+++ b/tools/meson.build
@@ -248,7 +248,7 @@ configure_file(
   configuration: tools_conf,
   install: true,
   install_dir: bindir,
-  install_mode: 'rwxrwxr-x',
+  install_mode: 'rwxr-xr-x',
 )
 
 configure_file(
@@ -257,7 +257,7 @@ configure_file(
   configuration: tools_conf,
   install: true,
   install_dir: bindir,
-  install_mode: 'rwxrwxr-x',
+  install_mode: 'rwxr-xr-x',
 )
 
 executable(
@@ -294,7 +294,7 @@ if conf.has('WITH_SANLOCK')
     configuration: tools_conf,
     install: true,
     install_dir: sbindir,
-    install_mode: 'rwxrwxr-x',
+    install_mode: 'rwxr-xr-x',
   )
 endif
 
